src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -699,11 +699,14 @@ message TaskKillRequest {
 }
 
 message TaskMoveRequest {
-    /// (Required) Task to move.
-    string taskId = 1;
+    /// (Required) Job which is the source of the task.
```

Comment:
We should document in `MoveTask` method the constraints for the source/target jobs (must be two different service jobs, task must be in the `Started` state, all fields identical except..., etc).